pac4j-core/src/main/java/org/pac4j/core/context/HttpConstants.java

```diff
@@ -42,6 +42,8 @@
 
     String AUTHORIZATION_HEADER = "Authorization";
 
+    String BASIC_HEADER_PREFIX = "Basic ";
```

Comment:
Let's get this merged. Looks pretty done. I ll work on the CAS side of things based on the snapshot here.